reactor-core/src/main/java/reactor/core/publisher/MonoOperator.java

```diff
@@ -54,4 +55,24 @@ public Object scanUnsafe(Attr key) {
 		return null;
 	}
 
+	@Override
+	@SuppressWarnings("unchecked")
```

Comment:
same comment here as in `FluxOperator`: decide whether or not to make this public API, and if so make subscribe `final`